### PR TITLE
remove unsupported feature from public document

### LINKED ISSUE #1496: Some sentences should to be deleted are remaining in the public document
### Describe the bug:

From users question, some sentences should to be deleted when `Simple Vald` is released.
However, these are remaining now.
e.x. https://vald.vdaas.org/doc/overview/about

### To Reproduce:

- Deleting the sentence which current Vald does not support from [these documents](https://github.com/vdaas/vald/tree/master/docs).

### Expected behavior:

- we should provide the public document as the official support document which contains exact support contents and does not contain unsupported contents.

### Environment:

- Go Version: 1.17.2
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.13.7

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

- SSIA

### Related Issue:

- close #1496 

### Environment:

- Go Version: 1.17.2
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.13.7

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Go to [Get Started](./docs/tutorial/get-started.md) page to try out Vald :)
     - Horizontal scalable on memory and CPU for your demand.
 
 - Auto Backup for Index data
-    - Vald has a feature to store the backup of the index data using MySQL or Cassandra which enables disaster recovery.
+    - Vald has a feature to store the backup of the each Vald Agent index data using Object Storage or Persistent Volume.
```
Comment (kevindiu) [inline README.md:50]: I think to simplify the sentences, we don't need to specify each Vald Agent.
```suggestion
    - Vald supports to backup Vald Agent index data using Object Storage or Persistent Volume.
```

File: docs/overview/about-vald.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Vald is easy to use, feature-rich and highly customizable as you needed.
     - Horizontal scalable on memory and CPU for your demand.
 
 - Auto Backup for Index data
-    - Vald has a feature to store the backup of the index data using MySQL or Cassandra which enables disaster recovery.
+    - Vald has a feature to store the backup of the each Vald Agent index data using Object Storage or Persistent Volume.
```
Comment (kevindiu) [inline docs/overview/about-vald.md:34]:
```suggestion
    - Vald supports to backup Vald Agent index data using Object Storage or Persistent Volume.
```

**Review by kevindiu — DISMISSED** [review]
LGTM

**Review by kmrmt — APPROVED** [review]
:+1:

**Review by hlts2 — DISMISSED** [review]

File: docs/user-guides/configuration.md [hunk 3]
```diff
@@ -208,9 +208,9 @@ agent:
 
 #### Ingress
 
-Ingress for gateways can be configured by `gateway.{backup,filter,lb,meta}.ingress` field object.
```
Comment (kpango) [inline docs/user-guides/configuration.md:211]: I think we shouldn't remove filter from Ingress section.

Comment (vankichi) [reply docs/user-guides/configuration.md:211]: oops, sorry, it's my mistake

File: docs/user-guides/configuration.md [hunk 4]
```diff
@@ -208,9 +208,9 @@ agent:
...
-It is important to set your host to `gateway.{backup,filter,lb,meta}.ingress.host` field.
-`gateway.{backup,filter,lb,meta}.ingress.servicePort` should be `grpc` or `rest`.
+Ingress for gateways can be configured by `gateway.lb.ingress` field object.
```
Comment (vankichi) [inline docs/user-guides/configuration.md:211]:
```suggestion
Ingress for gateways can be configured by `gateway.{filter,lb}.ingress` field object.
```

File: docs/user-guides/configuration.md (hunk 3 above)
Comment (vankichi) [reply docs/user-guides/configuration.md:211]: fixed :smile: 31f7d1a

**Review by kevindiu — APPROVED** [review]

**Review by kmrmt — APPROVED** [review]

File: docs/user-guides/configuration.md [hunk 5]
```diff
@@ -208,9 +208,9 @@ agent:
 
 #### Ingress
 
-Ingress for gateways can be configured by `gateway.{backup,filter,lb,meta}.ingress` field object.
-It is important to set your host to `gateway.{backup,filter,lb,meta}.ingress.host` field.
-`gateway.{backup,filter,lb,meta}.ingress.servicePort` should be `grpc` or `rest`.
+Ingress for gateways can be configured by `gateway.{filter,lb}.ingress` field object.
+It is important to set your host to `gateway.lb.ingress.host` field.
+`gateway.lb.ingress.servicePort` should be `grpc` or `rest`.
```
Comment (kevindiu) [inline docs/user-guides/configuration.md:213]:
```suggestion
It is important to set your host to `gateway.{filter,lb}.ingress.host` field.
`gateway.{filter,lb}.ingress.servicePort` should be `grpc` or `rest`.
```

**Review by kevindiu — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]

**Review by kevindiu — APPROVED** [review]

**Pull request merged**